File: platforms/iPad/Accounter iOS 1.0.2/Accounter/ResultList.m
//
//  ResultList.m
//  Accounter iOS2
//
//  Created by Amrit Mishra on 11/11/11.
//  Copyright 2011 __MyCompanyName__. All rights reserved.
//

#import "ResultList.h"


@implementation ResultList


-(id)init{
    
  
    
    self = [super init];
    if (self) {
        records = [[NSMutableArray alloc]init];
        title = [[NSString alloc]init];

    }
    return self;
}


- (void)setRecords:(NSMutableArray *)recordsList{
    
    records = [[NSMutableArray alloc]initWithArray:recordsList];
}

- (void)setMutiSelection:(BOOL)value{
    isMultiSelection = value;
}

- (void)setName:(NSString *)newName{
    name = newName;
}

- (NSString *)getName{
    return name;
}

- (NSMutableArray *)getRecords{
    return records;
}

- (BOOL)isMultiSelectionEnable{
    return isMultiSelection;
}

- (void)setTitle:(NSString *)newTitle{
    title = newTitle;
}

- (NSString *)getTitle{
    return title;
}

@end

File: platforms/blackberry/Accounter/src/com/vimukti/accounterbb/result/ResultList.java
package com.vimukti.accounterbb.result;

import java.util.Vector;

public class ResultList {
	private Vector records = new Vector();
	private boolean isMultiSelection;
	private String title;

	

	public void setMultiSelection(boolean isMultiSelection) {
		this.isMultiSelection = isMultiSelection;
	}

	public boolean isMultiSelection() {
		return isMultiSelection;
	}

	public void setRecords(Vector records) {
		this.records = records;
	}

	public Vector getRecords() {
		return records;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public String getTitle() {
		return title;
	}

	
}
